android/java/org/chromium/chrome/browser/rewards/BraveRewardsTippingPanelFragment.java

```diff
@@ -0,0 +1,535 @@
+    }
+
+    private void notEnoughFundLayout() {
+        ImageView imageView = sendDonationLayout.findViewById(R.id.send_tip_image);
+        sendDonationLayout.setBackgroundColor(ContextCompat.getColor(
+                getContext(), R.color.rewards_send_tip_not_enough_fund_background));
+        TextView sendDonationText = sendDonationLayout.findViewById(R.id.send_donation_text);
+        sendDonationText.setText(getResources().getString(R.string.not_enough_fund));
+        sendDonationText.setTextColor(ContextCompat.getColor(getContext(), android.R.color.white));
+        imageView.setImageResource(R.drawable.icn_frowning_face);
+        imageView.setColorFilter(ContextCompat.getColor(
+                getContext(), R.color.rewards_send_tip_icon_not_enough_fund_tint));
+        imageView.setVisibility(View.VISIBLE);
+    }
+
+    private void continueButtonLayout() {
+        ImageView imageView = sendDonationLayout.findViewById(R.id.send_tip_image);
+        sendDonationLayout.setBackgroundColor(
+                ContextCompat.getColor(getContext(), R.color.rewards_send_tip_background));
+        TextView sendDonationText = sendDonationLayout.findViewById(R.id.send_donation_text);
+        sendDonationText.setText(getResources().getString(R.string.continue_tip));
+        sendDonationText.setTextColor(ContextCompat.getColor(getContext(), android.R.color.white));
+        imageView.setImageResource(R.drawable.ic_send_tip);
+        imageView.setColorFilter(
+                ContextCompat.getColor(getContext(), R.color.rewards_send_tip_icon_tint));
+        imageView.setVisibility(View.GONE);
+    }
+
+    public void monthlyLayoutText() {
+        ImageView imageView = sendDonationLayout.findViewById(R.id.send_tip_image);
+        sendDonationLayout.setBackgroundColor(
+                ContextCompat.getColor(getContext(), R.color.rewards_send_tip_background));
+        TextView sendDonationText = sendDonationLayout.findViewById(R.id.send_donation_text);
+        sendDonationText.setText(getResources().getString(R.string.set_monthly_tip));
+        sendDonationText.setTextColor(ContextCompat.getColor(getContext(), android.R.color.white));
+        imageView.setImageResource(R.drawable.ic_calendar_icon);
+        imageView.setVisibility(View.VISIBLE);
+    }
+
+    public void resetSendLayoutText() {
+        ImageView imageView = sendDonationLayout.findViewById(R.id.send_tip_image);
+        sendDonationLayout.setBackgroundColor(
+                ContextCompat.getColor(getContext(), R.color.rewards_send_tip_background));
+        TextView sendDonationText = sendDonationLayout.findViewById(R.id.send_donation_text);
+        sendDonationText.setText(getResources().getString(R.string.send_tip));
+        sendDonationText.setTextColor(ContextCompat.getColor(getContext(), android.R.color.white));
+        imageView.setImageResource(R.drawable.ic_send_tip);
+        imageView.setVisibility(View.VISIBLE);
+    }
+}
```

Comment:
nit: new line and similar place at the end of every file

Reply:
done